### DISCUSSION TITLE
Need help deciding on a model

### DISCUSSION THREAD

**Discussion by Venkat-Shadeslayer** [mrdbourke/zero-to-mastery-ml#98] [Q&A]
Hi guys. Hope you all are having a great day today!
I am Venkat, a sophomore from IIT Madras, India and I am currently working on a classification problem.
The data is a 84X84 matrix which represents the brain connections of a human and we have 36 such samples.
We want to implement an ML model that classifies these 36 84X84 matrices into Control(C), Meningioma(M) or Glioma(G).

I am currently implementing an SVM-RGB kernal with Leave one out Cross Validation and I am getting an accuracy of about 60%.
I have used Optuna for finding out the optimal hyperparameters.

We have limited(36), high dimensional (84X84)  data points.

Could someone please share any ideas about what models to try out in order to bump up my accuracy?
Thanks a lot

**Comment by kRiShNa-429407** [comment]
Great!!

Have you tried with CatBoost and XGBoost??
